### ISSUE TITLE
a11y improvements on v2

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#1528]
Known issues:

* Back button on single problem view reads as 'back to all problem sets' - should read 'back to [Problem Set Title] set'

* Previous problem, next problem, add step buttons are not labelled accurately and should reflect their functions

* Three dot menus should be labelled as 'problem set options', 'step options' respectively

* Mathshare logo should receive focus before beta pill. Logo label should be 'Benetech Mathshare logo'

* On the single problem set card view, the math content on the individual problem cards does not read correctly. It reads as an edit box followed by each term. This rendering/a11y approach should mirror the approach in v1

* On the problem sets card view, the benetech globe logo should be labelled as 'created by benetech'

* on single problem view, these decorative icons should be hidden from screen reader - flag, pencil emoji, speech bubble emoji

* on single problem view, check the appproach for both input boxes. this should mirror the v1 approach for rendering these inputs. The math input reads edit box disabled, followed by an edit 'after' element.

* on single problem view, the prompt math is rendered in the same pattern as the problem set view. it needs to be the same approach as v1 (not edit box, sr version)

* on single problem view, the math inputs on steps with math entered is rendered in the same pattern as the prompt. it needs to be the same approach as v1 when the current content in the input gets read out allowing selection to edit

* Add headers across views. Mimic the heading structure in v1 for titles, cards, etc

Please review the a11y remediation documentation from v1 a11y review to ensure operation within those recommendations
